config/boards/orangepi4.conf

```diff
@@ -1,8 +1,9 @@
 # Rockchip RK3399 hexa core 4GB RAM SoC GBE eMMC USB3 USB-C WiFi/BT
 BOARD_NAME="OrangePi 4"
-BOARDFAMILY="rk3399"
+BOARDFAMILY="rockchip64"
```

Comment:
Why moving it to `rockchip64` family?
Does it have anything for Orange Pi 4 that `rk3399` lacks?
This move definitely borks `legacy` branch for Orange Pi 4

Reply:
ah is thats why he was in rk3399 

no it is just a rk3399 just like the rockchip64

did do dev no legacy but it not matter where it is it 

thnx